package/suc/run.sh

```diff
@@ -12,6 +12,8 @@ cleanup() {
     rm -rf "/host${TMPDIR}"
 }
 
+trap cleanup EXIT
```

Comment:
We may want to add a `trap exit INT HUP TERM` to catch signals as well, if we are enhancing the script this way.

Reply:
Do you mean update it like following?
```bash
trap cleanup EXIT
trap exit INT HUP TERM
```